index.html

```diff
@@ -84,38 +84,16 @@ <h1>
           <div>
             <h3>
               <img
-                src="./images/guidelines-icon.png"
-                alt="Representation icon for guidelines"
-              />
-              Guidelines
-            </h3>
-            <p>
-              The design guidelines have been curated by our design team in
-              order to help teams in building compelling and consistent User
-              Experiences.
-            </p>
-            <a
-              href="https://developer.dxc.com/design/guidelines/principles/overview"
-              >See guidelines and principles</a
-            >
```

Comment:
Maybe we should add a link to the halstack site, since we have the design guidelines there?
Maybe to this principles link https://developer.dxc.com/halstack/8/principles/color/.

Reply:
Done!